src/devices/controller.c

```diff
@@ -260,9 +148,16 @@ int32_t controller_print(controller_id_e_t id, uint8_t line, uint8_t col, const
 }
 
 int32_t controller_clear_line(controller_id_e_t id, uint8_t line) {
-	static const char* clear = "                   ";
-	kassert(strlen(clear) == CONTROLLER_MAX_COLS);
-	return controller_print(id, line, 0, clear);
+	if (vexSystemVersion() > 0x01000000) {
+		uint8_t port;
+		CONTROLLER_PORT_MUTEX_TAKE(id, port);
+		static char* clear = "";
+		vexControllerTextSet(id, ++line, 1, clear);
+		internal_port_mutex_give(port);
+		return 1;
+	} else {
+		return PROS_ERR;
+	}
```

Comment:
The vexSystemVersion is off. It should be whatever it was just updated to that fixed the clearing bug. For the case where we're on the older version of vex os and it's a case that's potentially buggy, I think we should call into the old version. It may not work the best, but it would prevent us from regressing currently working code if they don't update.

If you think it's too much effort for what it's worth, we can axe this idea, go back to what you had, and tell them to update vexos.

Reply:
Oooh ok I get it now. How do I indicate below version 1.12 though?

Reply:
See what vexSystemVersion returns on a 1.12 vexOS. My guess is something like 0x01C00000. If vexSystemVersion is below that, then that's what you're asking.